manager/test/api/src/test/resources/test-plan-data/clients/delete/002_delete-client-1.resttest

```diff
@@ -1,3 +1,3 @@
-DELETE /organizations/Organization1/clients/Client1 admin/admin
-----
-204
+DELETE /organizations/Organization1/clients/Client2 admin/admin
```

Comment:
Why did you change this plan to delete 'Client 2' instead of 'Client 1'?

Reply:
Because Client 2 does not have any Client-Versions. So I can check if Client 1 keep all Versions after the deletion of Client 2